x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -81,6 +81,7 @@ The following actions are possible and grouped based on the actions.
   KIBANA_FLEET_HOST - kibana host to enable create enrollment token on [$KIBANA_HOST]
   FLEET_TOKEN_NAME - token name to use for fetching token from Kibana. This requires Kibana configs to be set.
   FLEET_TOKEN_POLICY_NAME - token policy name to use for fetching token from Kibana. This requires Kibana configs to be set.
+  DEFAULT_FLEET_TOKEN_POLICY_NAME - token policy name to use when [$FLEET_TOKEN_POLICY_NAME] is not set. Defaults to "Default policy".
```

Comment:
Why is this extra needed? Can't the default just be there for `FLEET_TOKEN_POLICY_NAME` when one is not provided?

Reply:
this is in response to the above conversation saying the default should be configurable. maybe i misunderstood what was being said?

it does seem a bit redundant to me though, since someone could just set the policy id up front when they are configuring their instance.

i suppose it _could_ be useful if someone wanted to set up different defaults assuming the policy an agent is looking for disappears. not too sure how much we want to worry about that situation.